easybuild/easyconfigs/s/SIP/SIP-4.19.25-foss-2022a-Python-3.10.4.eb

```
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Bart Verleye
+# Center for eResearch, Auckland
+easyblock = 'ConfigureMakePythonPackage'
+
+name = 'SIP'
+version = '4.19.25'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.riverbankcomputing.com/software/sip/'
+description = """SIP is a tool that makes it very easy to create Python bindings for C and C++ libraries."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+source_urls = ['http://sourceforge.net/projects/pyqt/files/sip/sip-%(version)s']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['b39d93e937647807bac23579edbff25fe46d16213f708370072574ab1f1b4211']
+dependencies = [('Python', '3.10.4')]
```

Very nitpicky, but we usually have a blank line here, as dependencies does not really belong to the source items
```suggestion

dependencies = [('Python', '3.10.4')]
```